wordpress-mobile/WordPress-Android
# [Pre-publish] Pre-publish sheet improvements

Fixes #18975 

Change some text typography, divider position/visibility, and some margins to match improvement specs from "Jetpack Social" Figma project.

To test:
Ideally, have a site with no JP Social connections, one with limited sharing and connections (SH with JP Social plugin), and one WPCOM with connections (unlimited).

1. Open the Jetpack app
2. Select the site
3. Go to Posts
4. Create a new Post
5. Hit `Publish`
6. **Verify** the styles are updated

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
UI only.

PR submission checklist:


UI Changes testing checklist:
